app/client/src/components/shared/AddSaveDataWidget.js

```diff
@@ -127,18 +127,20 @@ const layerPanelStyles = (visible) => css`
   display: ${visible ? 'flex' : 'none'};
   flex-flow: column;
   height: 100%;
-`;
 
-const layerPanelHeaderStyles = css`
-  height: 40px;
-  margin: 0;
-  padding: 7px 15px;
-  border-bottom: 1px solid #ccc;
-  color: #898989;
-  font-size: 16px;
-  font-weight: normal;
+  h3 {
+    font-family: ${fonts.primary};
+    margin: 0;
+    padding: 7px 15px;
+    border-bottom: 1px solid #ccc;
+    color: #898989;
+    font-size: 16px;
+    font-weight: normal;
+  }
 `;
 
+const layerPanelHeaderStyles = css``;
+
```

Comment:
```suggestion
```

This can be removed as it is no longer used.

Reply:
I went ahead and removed it and commited the change.

Reply:
Thank you.